src/test/java/com/hocs/test/pages/teamqueue/Teamqueue.java

```diff
@@ -0,0 +1,137 @@
+package com.hocs.test.pages.teamqueue;
+
+import static net.serenitybdd.core.Serenity.sessionVariableCalled;
+import static net.thucydides.core.pages.components.HtmlTable.rowsFrom;
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.containsString;
+import static org.hamcrest.Matchers.hasItem;
+import static org.hamcrest.core.Is.is;
+
+import com.hocs.test.pages.Page;
+import java.util.List;
+import java.util.Map;
+import net.serenitybdd.core.annotations.findby.FindBy;
+import net.serenitybdd.core.pages.WebElementFacade;
+import net.thucydides.core.annotations.Managed;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+
+public class Teamqueue extends Page {
+
+    @Managed
+    WebDriver driver;
```

Comment:
you should be able to call getDriver() anywhere on a page to get the current working driver.